assets/src/components/selection/event-select/index.js

```diff
@@ -101,6 +103,9 @@ EventSelect.defaultProps = {
 		order: 'desc',
 		showExpired: false,
 	},
+	selectLabel: __( 'Select Event', 'event_espresso' ),
+	isLoading: true,
+	events: [],
```

Comment:
I moved the defaults here instead of in the constructor for the component (since I was already defining defaults here).